### DISCUSSION TITLE
The projects that we need to research sooner or later

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#47] [General]
![ecosystem](https://user-images.githubusercontent.com/49662536/166322303-9908971f-0672-4699-bb8a-dd77ef133796.jpeg)
[Source](https://twitter.com/Algo1circle/status/1521128167699894272)

Use this thread to enforce some of your favorites, or at least bring some reasons why we should add them sooner. :)

**Comment by lario0913** [comment]
Just curious on how NF Domain will be integrated as there can be conflict of domain with ANS 🤔

**Reply by dragmz** [reply]
The ideas so far are to handle with custom prefix / suffix and/or resolve with all available name services to let user pick the right one in the UI.

**Reply by lario0913** [reply]
That's good then
